src/index.js

```diff
@@ -22,27 +23,25 @@ export default function RollupPluginPreprocess ({
     name: 'preprocess',
 
     /**
-     * @param {string} code
      * @param {string} fileName
-     * @returns {{ code: string, map: Object }}
+     * @returns {string}
      */
-    transform (code, fileName) {
-      if (!filter(fileName)) {
-        return;
-      }
+    load (fileName) {
+      let data = fs.readFileSync(fileName);
```

Comment:
Having issues with this loading **.less** files. `readFileSync` returns a Buffer and Rollup throws the error:

```
Error: Error loading src/Test.less: plugin load hook should return
a string, a { code, map } object, or nothing/null
```

When I changed to:
```js
let data = fs.readFileSync(fileName, 'utf8');
```
It worked fine.